spec/azure/virtual_network.go

```diff
@@ -0,0 +1,7 @@
+package azure
+
+type VirtualNetwork struct {
+	CIDR             string `json:"cidr" yaml:"cidr"`
+	MasterSubnetCIDR string `json:"masterSubnetCidr" yaml:"masterSubnetCidr"`
```

Comment:
I'll update the templates (https://github.com/giantswarm/azure-operator/tree/main-arm-template) so they're using the same as you define here (e.g. masterSubnetCidr instead of subnetMastersCIDR)